cmd/cdi-importer/importer.go

```diff
@@ -56,10 +59,27 @@ func main() {
 		contentType,
 		imageSize,
 	}
-	err = importer.CopyImage(dso)
-	if err != nil {
-		glog.Errorf("%+v", err)
-		os.Exit(1)
+
+	if source == controller.SourceNone && contentType == controller.ContentTypeKubevirt {
+		qImageSize := resource.MustParse(imageSize)
```

Comment:
Doesn't MustParse produce a quantity already, why are we changing it to a string and then back to a quantity again?